scripts/core/helpers/waitForMediaToBeReady.ts

```diff
@@ -1,37 +1,34 @@
 /**
- * @param {Array<Node>} elements
- * @param {string} eventName
+ * @param elements An array of img, audio and video elements
+ * @param eventName EventListener's name
  */
-const waitForEventToFireForAllElements = (elements, eventName) => new Promise((resolve) => {
-    let itemsLeftToLoad = elements.length;
+export const waitForMediaToLoad = (elements: Array<any>, eventName: string): Promise<void> => new Promise((resolve) => {
```

Comment:
Don't use `any` type when the actual type is known. It should be `elements: Array<Element>`.